distributions/raspijamulus.sh

```diff
@@ -6,11 +6,11 @@ NCORES=$(nproc)
 
 # install required packages
 pkgs='alsamixergui build-essential qtbase5-dev qttools5-dev-tools libasound2-dev cmake libglib2.0-dev'
-if ! dpkg -s $pkgs >/dev/null 2>&1; then
+if ! dpkg -s "${pkgs}" >/dev/null 2>&1; then
   read -p "Do you want to install missing packages? " -n 1 -r
   echo
-  if [[ $REPLY =~ ^[Yy]$ ]]; then
-    sudo apt-get install $pkgs -y
+  if [[ ${REPLY} =~ ^[Yy]$ ]]; then
+    sudo apt-get install "${pkgs}" -y
```

Comment:
I don't think this can work:
```
root@e7169628f80d:/# x="vim screen"; apt install "$x"
[...]
E: Unable to locate package vim screen
```

This should either be kept unquoted or use the proper expansion syntax (not sure if I remember correctly, but it could have been `"${pkgs[@]}"` when the pkgs definition would be changed into an array).

Reply:
Yes, you're going to have to use an array if any element of `pkgs` could have a space in the name and there are multiple arguments.  Never use one variable to represent multiple arguments.

Reply:
But this doesn't apply when we know what is in `pkgs` because it was set explicitly in line 8.

Reply:
Line 8 is still ugly bash script.  It would be better as `pkgs=(name0 name1 ...)` and then `"${pkgs[@]}` to properly expand it.  Relying on not having spaces in order for expansion to work _is_ bad coding.